### PR TITLE
Feature sidebar messages

### PR DESCRIPTION
**Pull request by tslekwadi**

### CODE REVIEW

File: src/containers/sidebar/index.js [hunk 1]
```diff
@@ -13,62 +13,39 @@ import Chip from 'material-ui/Chip';
...
 import './index.css';
-//import 'typeface-roboto';
+import MessagesContainer from "./messagesContainer"
 
-const Sidebar = () => (
-    <div class="SidebarParent">
-        <MuiThemeProvider>
+export default class Sidebar extends React.Component
+{
+    constructor()
```
Comment (KimberleighBeyl) [inline src/containers/sidebar/index.js]: not needed

File: src/containers/sidebar/messagesContainer.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import SearchBar from 'material-ui-search-bar';
+import Chip from 'material-ui/Chip';
+import Avi from './avi.jpg';
+import Divider from 'material-ui/Divider';
+import './index.css';
+
+export default class MessagesContainer extends React.Component
+{
+    
+    constructor()
```
Comment (KimberleighBeyl) [inline src/containers/sidebar/messagesContainer.js]: again if you don't send through props delete constructor

**Review by KimberleighBeyl — CHANGES_REQUESTED** [review]
Please just check whether you want to send through props in the constructor

File: src/containers/sidebar/messagesContainer.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    {
+        super();
+    }
+
+    //render method
```
Comment (wvictor-rr) [inline src/containers/sidebar/messagesContainer.js]: Unnecessary comment

**Review by KimberleighBeyl — APPROVED** [review]
:+1:

**Pull request merged**